.gitignore

```diff
@@ -0,0 +1,4 @@
+﻿.idea
+go.sum
+go.sum
+*.sum
```

Comment:
This makes the rules on lines 2 and 3 extraneous. In any event, I'm not sure it's what we want.

Reply:
I don't understand, maybe should I remove the .gitignore altogether from the versioning?
I now notice that this repo doesn't have one.